### DISCUSSION TITLE
Error in webpack-stream: PluginError:

### DISCUSSION THREAD

**Discussion by tantian1498** [webpack/webpack#18455] [Q&A]
An error occurred when upgrading from version 4 to version 5, but the specific error location cannot be located. What should I do:

{
moduleIdentifier: 'webpack/runtime/compat',
moduleName: 'webpack/runtime/compat',
message: 'undefined is not iterable (cannot read property Symbol(Symbol.iterator))',
moduleTrace: []
},
{
moduleIdentifier: 'webpack/runtime/compat',
moduleName: 'webpack/runtime/compat',
message: 'undefined is not iterable (cannot read property Symbol(Symbol.iterator))',
moduleTrace: []
}
]
Error in webpack-stream: PluginError:

**Comment by alexander-akait** [comment]
Please create reproducible test repo using github, thank you
